.travis.yml

```diff
@@ -55,6 +55,11 @@ jobs:
         - IOS_BUILD_TYPE='Debug'
         - IOS_PLATFORM_TYPE='iphonesimulator'
         - DST='platform=iOS Simulator,OS=13.2.2,name=iPhone 11'
+      script:
+        - scripts/ci/build.sh --build MapboxVision
```

Comment:
I think the tests will build `MapboxVision` as a dependency, there is no need to waste time on a standalone build